Sail/src/Sail/entities/systems/Gameplay/ai/AiSystem.cpp

```diff
@@ -129,6 +129,13 @@ NodeSystem* AiSystem::getNodeSystem() {
 	return m_nodeSystem.get();
 }
 
+#ifdef DEVELOPMENT
+unsigned int AiSystem::getByteSize() const {
+	unsigned int size = sizeof(*this);
+	size += sizeof(NodeSystem);
+	return size;
```

Comment:
Size of node system needs to be reflected as well since it has several items inside of it not reflected by sizeof(NodeSystem)

Reply:
Fixed, thanks for the reminder